test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/DataMemberBindingMetadataProviderTest.cs

```diff
@@ -1,96 +0,0 @@
-// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Runtime.Serialization;
-using Xunit;
-
-namespace Microsoft.AspNet.Mvc.ModelBinding.Metadata
-{
-    public class DataMemberBindingMetadataProviderTest
-    {
-        [Fact]
-        public void ClassWithoutAttributes_NotRequired()
```

Comment:
Moved these existing tests to `ModeMetadataProviderTest` because they are actually integration tests. Added new unit tests to the renamed version of this file which cover the same cases.